content/posts/2017-05-03-normandy-and-auth0.mkd

```diff
@@ -1,20 +1,20 @@
-title: Switching Normandy to use Auth0
+title: Switching Normandy to use OIDC
```

Comment:
OIDC is the protocol we're using and Auth0 is a company, a service, and also the name of the software that Mozilla runs on our SSO. Nothing we're doing is Auth0-specific, so I think it's better to say OIDC here (and in other places).